packages/corvid-fake-local-mode-editor/src/browser/server.js

```diff
@@ -9,7 +11,7 @@ const bundle = browserify(path.resolve(path.join(__dirname, "browser.js")));
 
 const runningServers = [];
 
-function start() {
+function start(extraArgs = "") {
```

Comment:
Maybe try to convert extraArgs from string to an object/JSON

Reply:
done